README.md

```diff
@@ -1,3 +1,28 @@
-## About
+# About
 
-This is a (very) lightweight discord clone.
+Liscord is a (very) lightweight discord clone.
+
+# Requirements
+
+* Python 3.x
+
+# Installation
+
+* git clone https://github.com/raqibahmed/liscord
+
+```bash
+pip install -r requirements.txt
+```
+```bash
+pip install phonenumbers
+```
+```bash
+.manage.py runserver
```

Comment:
I think you missed a slash. The correct command will be `./manage.py runserver` or `python manage.py runserver`